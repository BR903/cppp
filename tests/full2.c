#if (defined(foo) || defined bar)
one
#elif (!defined(foo) && !defined(baz))  // comment
two
#elif defined foo ? !defined bar : !defined baz
three
#endif
