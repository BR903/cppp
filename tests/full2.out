/* -Dfoo */
one
/* -Dbar */
one
/* -Dbaz */
#if (defined(foo) || defined bar)
one
#elif defined foo ? !defined bar : 0
three
#endif
/* -Ufoo */
#if defined bar
one
#elif !defined(baz)  // comment
two
#elif !defined baz
three
#endif
/* -Ubar */
#if defined(foo)
one
#elif (!defined(foo) && !defined(baz))  // comment
two
#elif defined foo ? 1 : !defined baz
three
#endif
/* -Ubaz */
#if (defined(foo) || defined bar)
one
#elif !defined(foo)  // comment
two
#elif defined foo ? !defined bar : 1
three
#endif
/* -Dfoo -Dbar */
one
/* -Dfoo -Dbaz */
one
/* -Dbar -Dbaz */
one
/* -Dfoo -Dbar -Dbaz */
one
/* -Ufoo -Ubar */
#if !defined(baz)  // comment
two
#elif !defined baz
three
#endif
/* -Ufoo -Ubaz */
#if defined bar
one
#else
two
#endif
/* -Ubar -Ubaz */
#if defined(foo)
one
#elif !defined(foo)  // comment
two
#else
three
#endif
/* -Ufoo -Ubar -Ubaz */
two
/* -Dfoo -Ubar */
one
/* -Dfoo -Ubaz */
one
/* -Dbar -Ubaz */
one
/* -Dfoo -Ubar -Ubaz */
one
/* -Ufoo -Dbar */
one
/* -Ufoo -Dbaz */
#if defined bar
one
#endif
/* -Ubar -Dbaz */
#if defined(foo)
one
#elif defined foo ? 1 : 0
three
#endif
/* -Ufoo -Dbar -Dbaz */
one
/* -Dfoo -Ubar -Dbaz */
one
/* -Ufoo -Ubar -Dbaz */
/* -Ufoo -Dbar -Ubaz */
one
/* -Dfoo -Dbar -Ubaz */
one
